src/nni_manager/training_service/kubeflow/kubeflowConfig.ts

```diff
@@ -76,6 +80,36 @@ export class NFSConfig {
     }
 }
 
+/**
+ * KeyVault configuration to store the key of Azure Storage Service
+ */
+export class keyVaultConfig {
+    /**The vault-name to specify vault */
+    public readonly vaultName : string;
+    /**The name to specify private key */
+    public readonly name : string;
```

Comment:
What does this field mean? 
If it's a private key, rename it to private key. 

Your naming for keyVaultConfig variables SHOULD be consistent with Azure official defination for key vault